test/e2e/specs/multi-block-selection.test.js

```diff
@@ -17,7 +15,7 @@ describe( 'Multi-block selection', () => {
 		const multiSelectedCssClass = 'is-multi-selected';
 
 		// Creating test blocks
-		await page.click( '.editor-default-block-appender' );
+		await page.click( '.editor-default-block-appender__content' );
```

Comment:
Just curious why this change?

Reply:
For consistency, we probably should wrap it with a utility method. Initially, I suspected that it might be the reason why undo test fails sometimes.

Reply:
This updated class is the one which has the events bound, so it's more "correct".